apps/src/templates/sectionProgress/SectionProgress.jsx

```diff
@@ -282,12 +229,7 @@ export default connect(
     currentView: state.sectionProgress.currentView,
     scriptData: getCurrentScriptData(state),
     isLoadingProgress: state.sectionProgress.isLoadingProgress,
-    showStandardsIntroDialog: !state.currentUser.hasSeenStandardsReportInfo,
-    studentTimestamps:
```

Comment:
these props were only needed for the tooltips, which have been relocated per the above comment.